apps/hash-frontend/src/pages/account/integrations/use-linear-integrations.ts

```diff
@@ -0,0 +1,97 @@
+import { types } from "@local/hash-isomorphic-utils/ontology-types";
+import { Entity } from "@local/hash-subgraph/.";
+import {
+  getOutgoingLinkAndTargetEntities,
+  getRoots,
+} from "@local/hash-subgraph/stdlib";
+import { extractBaseUrl } from "@local/hash-subgraph/type-system-patch";
+import { useEffect, useState } from "react";
+
+import { useBlockProtocolQueryEntities } from "../../../components/hooks/block-protocol-functions/knowledge/use-block-protocol-query-entities";
+import { useAuthenticatedUser } from "../../shared/auth-info-context";
+
+export type LinearIntegration = {
+  entity: Entity;
+  syncedWithWorkspaces: { workspaceEntity: Entity; linearTeamIds: string[] }[];
+};
+
+export const useLinearIntegrations = () => {
+  const { authenticatedUser } = useAuthenticatedUser();
+  const { queryEntities } = useBlockProtocolQueryEntities();
+
+  const [linearIntegrations, setLinearIntegrations] =
+    useState<LinearIntegration[]>();
+
+  useEffect(() => {
+    void (async () => {
+      const { data: subgraph } = await queryEntities({
+        data: {
+          operation: {
+            multiFilter: {
+              filters: [
+                /** @todo: figure out how to make this work */
+                // {
+                //   field: ["ownedById"],
+                //   operator: "EQUALS",
+                //   value: authenticatedUser.accountId,
+                // },
```

Comment:
I think this should work now

Reply:
Yes, thanks for reminding. Have made use of it in https://github.com/hashintel/hash/pull/2826/commits/636dff47faad60a18951019aba7e863afdc6f0b0

Reply:
Was there any client-side filtering that's now redundant?

Reply:
There wasn't a client side filter in this instance, no. I think I removed it when moving the query to the hook, or may have left it out entirely.